source/go-perun/index.rst

```diff
@@ -0,0 +1,48 @@
+Go-Perun
+========
+
```

Comment:
Could we add a short introduction to `go-perun` before the `developer tutorial` section ?

Reply:
Yes, this was also planned. But it needs some time and I propose to do it in another PR.